### PR TITLE
Add permissions instructions for the data volume

### PR DESCRIPTION
**Pull request by Angristan**

### CODE REVIEW

File: support/doc/docker.md [hunk 1]
```diff
@@ -50,6 +50,19 @@ balancer, although any HTTP reverse proxy will work fine. See the example
...
 recommendations and requirements to run PeerTube the most efficiently.
 
+
+When starting the containers for the first time, you will get permissions errors for the data volume, like this one :
+
+```
+Error: EACCES: permission denied, mkdir '/data/logs'
+```
+
+The peertube user inside the container has a UID and GID of 991 so you have the chown the folder, in the case your using `./data`:
```
Comment (bnjbvr) [inline support/doc/docker.md]: nit: `the chown` => `to chown`
`in the case your` => `in case you're`

File: support/doc/docker.md [hunk 2]
```diff
@@ -50,6 +50,19 @@ balancer, although any HTTP reverse proxy will work fine. See the example
 Nginx configuration `support/nginx/peertube` file to get an idea of
 recommendations and requirements to run PeerTube the most efficiently.
 
+
+When starting the containers for the first time, you will get permissions errors for the data volume, like this one :
```
Comment (bnjbvr) [inline support/doc/docker.md]: nit: no spaces before `:`

File: support/doc/docker.md (hunk 1 above)
Comment (bnjbvr) [reply support/doc/docker.md]: Also, I'd expand `chown the folder` to `change the folder's owner`

**Comment by Angristan** [comment]
Sorry I was a bit tired because I was trying to get Peertube to work the night before o/

Also I... deleted the branch by mistake, so let's continue in https://github.com/Chocobozzz/PeerTube/pull/387 (sorry)

**Pull request closed without merging**